app.js

```diff
@@ -267,3 +267,29 @@ contactForm.addEventListener('submit', (e) => {
     }
   }
 });
+
+if (localStorage == null) {
+  const formData = {
+    user_name: '',
+    user_email: '',
+    user_message: '',
+  };
+  localStorage.setItem('myFormData', JSON.stringify(formData));
+}
+
+const name = document.querySelector('#user-name');
+const email = document.querySelector('#contact-email');
+const message = document.querySelector('#msg');
+const savedData = JSON.parse(localStorage.getItem('myFormData'));
+// read data from local storage if any
+name.value = savedData.user_name;
+email.value = savedData.user_email;
+message.value = savedData.user_message;
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ Required ] Good job preserving the data in the browser, however, I think your project will be much improved if you don't disable linters from your written code. If you have to disable linters in your project it's better if you do it in the  configuration files e.g `.eslintrc.json` file. Please see the example below for more details on this.

I have added  `no-unused-vars: "off"`, `next-line: "off"` to the `.eslintrc.json` file to achieve the same thing you have done in your code. 
- [Optional ] I would also like to mention that disabling linters is not considered good practice it would be much better if you are able to solve the linter error without disabling it.

## .eslintrc.json
```JavaScript
{
  "env": {
    "browser": true,
    "es6": true,
    "jest": true
  },
  "parser": "babel-eslint",
  "parserOptions": {
    "ecmaVersion": 2018,
    "sourceType": "module"
  },
  "extends": ["airbnb-base"],
  "rules": {
    "no-shadow": "off",
    "no-param-reassign": "off",
    "eol-last": "off",
    "no-unused-vars": "off",
    "next-line": "off",
    "import/extensions": [ 1, {
      "js": "always", "json": "always"
    }]
  },
  "ignorePatterns": [
    "dist/",
    "build/"
  ]
}

```